test/topology_custom/test_tablets.py

```diff
@@ -166,6 +166,35 @@ async def check_allocated_replica(expected: int):
         assert len(fragments[k]) == rf_to, f"Found mutations for {k} key on {fragments[k]} hosts, but expected only {rf_to} of them"
 
 
+# ALTER tablets KS cannot change RF of any DC by more than 1 at a time.
+# In a multi-dc environment, we can create replicas in a DC that didn't have replicas before,
+# but the above requirement should still be honoured, because we'd be changing RF from 0 to N in the new DC.
+# Reproduces https://github.com/scylladb/scylladb/issues/20039#issuecomment-2271365060
+# See also cql-pytest/test_tablets.py::test_alter_tablet_keyspace_rf for basic scenarios tested
+@pytest.mark.asyncio
+async def test_multidc_alter_tablets_rf(request: pytest.FixtureRequest, manager: ManagerClient) -> None:
+    config = {"endpoint_snitch": "GossipingPropertyFileSnitch", "enable_tablets": "true"}
+
+    logger.info("Creating a new cluster of 1 node in 1st DC and 2 nodes in 2nd DC")
+    await manager.server_add(config=config, property_file={'dc': f'dc1', 'rack': 'myrack'})
+    await manager.server_add(config=config, property_file={'dc': f'dc2', 'rack': 'myrack'})
+    await manager.server_add(config=config, property_file={'dc': f'dc2', 'rack': 'myrack'})
+
+    conn = manager.get_cql()
+    conn.execute("create keyspace ks with replication = {'class': 'NetworkTopologyStrategy', 'dc1': 1};")
```

Comment:
`execute` -> `await ... run_async`. `run_async` is defined to choose timeouts to avoid test flakiness.

Reply:
Fixed.

Reply:
It should be fixed in the commit where you add the test.

Reply:
@ptrsmrn - please resolve comments, if relevant.

Reply:
Fixed.